frontend/pages/features/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from "react";
```

Comment:
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/61459744/169291836-9a6bb521-e9bf-42b7-aab6-c88a222b2b21.png">
Features page padding looks too small in all screen sizes below ~1420px